go.mod

```diff
@@ -4,71 +4,3 @@
 module ms-authentication
 
 go 1.18
```

Comment:
not sure why you have to modify files outside of ds-cv-inference, but if you do, you should upgrade the go version to 2.0

Reply:
let me restore these.